### DISCUSSION TITLE
My device can not find each other

### DISCUSSION THREAD

**Discussion by Ericus55** [localsend/localsend#1193] [Discussion (English, International)]
My laptop and phone can not find each other anymore. They used to be in the same network but now, even though they both use the same wifi, they can not find each other. What has happened? They both have different IP addresses and I can´t change them.

**Comment by gamekaka505** [comment]
Am also facing the same trouble. When am at someone else' house and am using their wifi network, the file transfer from my device to the computer makes it very easy transfer but when am at my home network, the pc wont find the phone

**Comment by ykykle** [comment]
我也遇到了一样的问题，我的两个手机和平板之间都无法发现对方，我特意下载了电脑端，可还是不行

**Comment by RiskySolutions** [comment]
could be a permissions issues

**Comment by czfadmin** [comment]
需要对一方的app进行关闭然后重新打开另一方应用才可以显示, 这应该是个BUG,我在ios上遇到这个问题. 将app置于后台一段时间,然后电脑端会发送失败,或者扫描不到问题, 一般都是关闭销毁app,然后重新打开app就可以正常使用了
You need to close one app and then reopen the other app before it can be displayed. This should be a BUG. I encountered this problem on iOS. Put the app in the background for a period of time, and then the computer will fail to send, or the scan cannot be done. Problem, usually you close and destroy the app, then reopen the app and you can use it normally.
